plugins/plugin-chart-echarts/src/Timeseries/transformers.ts

```diff
@@ -141,10 +162,29 @@ export function transformSeries(
           ? opacity * areaOpacity
           : 0,
     },
-    showSymbol:
-      !isConfidenceBand &&
-      (plotType === 'scatter' || (forecastEnabled && isObservation) || markerEnabled),
+    showSymbol,
     symbolSize: markerSize,
+    label: {
+      show: !!showValue,
+      position: 'top',
+      formatter: (params: any) => {
+        const {
+          value: [_, numericValue],
+          dataIndex,
+          seriesIndex,
+        } = params;
+        if (formatter) {
+          if (!stack) {
+            return formatter(numericValue);
+          }
+          if (seriesIndex === showValueIndexes[dataIndex]) {
```

Comment:
Here is the key to show the stack value, only if the series index is equal to our `showValueIndexes` will we show the value, otherwise we return the empty string.